src/main/java/com/opentable/db/postgres/embedded/EmbeddedPostgres.java

```diff
@@ -73,6 +71,7 @@
 import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
 
 @SuppressWarnings("PMD.AvoidDuplicateLiterals") // "postgres"
+@SuppressFBWarnings({"RCN_REDUNDANT_NULLCHECK_OF_NONNULL_VALUE", "RCN_REDUNDANT_NULLCHECK_WOULD_HAVE_BEEN_A_NPE"}) // java 11 triggers: https://github.com/spotbugs/spotbugs/issues/756
```

Comment:
I'm pretty close to dropping spotbugs myself - it seems clear they just don't have the brainpower to fix these issues. Your thoughts?

Reply:
Unfortunately my projects all run `spotbugs` in jdk8 with `maven-toolchain-plugin` due to problems like these.  I think it's still worthwhile but it similarly bugs me that they can't get their jdk11 shit together